docs/architecture/README.md

```diff
@@ -118,3 +118,4 @@ None
 - [ADR-071: Proposer-Based Timestamps](./adr-071-proposer-based-timestamps.md)
 - [ADR-073: Adopt LibP2P](./adr-073-libp2p.md)
 - [ADR-074: Migrate Timeout Parameters to Consensus Parameters](./adr-074-timeout-params.md)
+- [ADR-080: Libp2p Implementation](./adr-080-libp2p-implementation.md)
```

Comment:
80 & 81 are already in-use (see below). (I'm sitting on 82 but that's still technically unclaimed for now 😀)

Reply:
you can have 83 :)